public/app/core/utils/explore.test.ts

```diff
@@ -440,3 +458,28 @@ describe('when buildQueryTransaction', () => {
     expect(transaction.request.interval).toEqual('2h');
   });
 });
+
+describe('generateEmptyQuery', () => {
+  it('should generate query with dataSourceOverride and without queries', async () => {
```

Comment:
`datasourceOverride` is internal implementation detail, I'd rather call it "with mixed data source".